text/0085-api-keys.md

```diff
@@ -368,8 +368,67 @@ Only the master key allows managing the API keys.
 ```
 "Authorization: Bearer :masterKey"
 ```
+##### Query Parameters
 
-##### Response
+| Field                    | Type                     | Required |
+|--------------------------|--------------------------|----------|
+| `offset`                 | Integer / `null`         | false    |
+| `limit`                  | Integer / `null`         | false    |
```

Comment:
So, if I understand it correctly, we're basically enabling pagination of API key GET requests, right?

Obviously not opposed in any way, but, out of curiosity, do we expect people to use so many keys they'll need pagination?

Reply:
Hi @guimachiavelli 👋

Exactly, it is to avoid a future breaking if we change later the number of results by adding the pagination, it also to be consistent by having pagination on all the lists of resources of the API from v0.28

It's possible that this could happen, especially for a company that would decide to have several indexes with specific keys per operation for each index for some security reasons / internal organization. 

We haven't had any expression of this need but the cloud team might also want to paginate to improve the UX of the dashboard at some point.

A User who creates no more than 20 keys shouldn't be directly bothered. The API keys are still in the `results` array